<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="11" passed="9" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-15T17:45:44 EDT" name="Suite" finished-at="2023-05-15T17:46:05 EDT" duration-ms="20756">
    <groups>
    </groups>
    <test started-at="2023-05-15T17:45:44 EDT" name="Test1" finished-at="2023-05-15T17:45:50 EDT" duration-ms="5039">
      <class name="baseClasses.SecondTest">
        <test-method signature="seond()[pri:0, instance:baseClasses.SecondTest@783a467b]" started-at="2023-05-15T17:45:50 EDT" name="seond" finished-at="2023-05-15T17:45:50 EDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seond -->
      </class> <!-- baseClasses.SecondTest -->
      <class name="baseClasses.FirstTest">
        <test-method signature="first()[pri:0, instance:baseClasses.FirstTest@4b0d79fc]" started-at="2023-05-15T17:45:45 EDT" name="first" finished-at="2023-05-15T17:45:50 EDT" duration-ms="5006" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- first -->
      </class> <!-- baseClasses.FirstTest -->
    </test> <!-- Test1 -->
    <test started-at="2023-05-15T17:45:50 EDT" name="Test2" finished-at="2023-05-15T17:45:55 EDT" duration-ms="5017">
      <class name="baseClasses.SecondTest">
        <test-method signature="seond()[pri:0, instance:baseClasses.SecondTest@6e6f2380]" started-at="2023-05-15T17:45:55 EDT" name="seond" finished-at="2023-05-15T17:45:55 EDT" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seond -->
      </class> <!-- baseClasses.SecondTest -->
      <class name="baseClasses.FirstTest">
        <test-method signature="first()[pri:0, instance:baseClasses.FirstTest@6892b3b6]" started-at="2023-05-15T17:45:50 EDT" name="first" finished-at="2023-05-15T17:45:55 EDT" duration-ms="5003" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- first -->
      </class> <!-- baseClasses.FirstTest -->
      <class name="baseClasses.ThirdTest">
        <test-method signature="third()[pri:0, instance:baseClasses.ThirdTest@76a4ebf2]" started-at="2023-05-15T17:45:55 EDT" name="third" finished-at="2023-05-15T17:45:55 EDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- third -->
      </class> <!-- baseClasses.ThirdTest -->
    </test> <!-- Test2 -->
    <test started-at="2023-05-15T17:45:55 EDT" name="Test3" finished-at="2023-05-15T17:45:58 EDT" duration-ms="3779">
      <class name="baseClasses.FBPageTest">
        <test-method signature="demoTest()[pri:0, instance:baseClasses.FBPageTest@8f2ef19]" started-at="2023-05-15T17:45:55 EDT" name="demoTest" finished-at="2023-05-15T17:45:58 EDT" duration-ms="3773" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Amsar] but found [Amar]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Amsar] but found [Amar]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at baseClasses.FBPageTest.demoTest(FBPageTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoTest -->
      </class> <!-- baseClasses.FBPageTest -->
      <class name="baseClasses.ThirdTest">
        <test-method signature="third()[pri:0, instance:baseClasses.ThirdTest@2eae8e6e]" started-at="2023-05-15T17:45:55 EDT" name="third" finished-at="2023-05-15T17:45:55 EDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- third -->
      </class> <!-- baseClasses.ThirdTest -->
    </test> <!-- Test3 -->
    <test started-at="2023-05-15T17:45:58 EDT" name="Test4" finished-at="2023-05-15T17:46:05 EDT" duration-ms="6884">
      <class name="baseClasses.FBPageTest">
        <test-method signature="demoTest()[pri:0, instance:baseClasses.FBPageTest@74ea2410]" started-at="2023-05-15T17:46:03 EDT" name="demoTest" finished-at="2023-05-15T17:46:05 EDT" duration-ms="1865" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Amsar] but found [Amar]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Amsar] but found [Amar]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at baseClasses.FBPageTest.demoTest(FBPageTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoTest -->
      </class> <!-- baseClasses.FBPageTest -->
      <class name="baseClasses.SecondTest">
        <test-method signature="seond()[pri:0, instance:baseClasses.SecondTest@173ed316]" started-at="2023-05-15T17:46:03 EDT" name="seond" finished-at="2023-05-15T17:46:03 EDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seond -->
      </class> <!-- baseClasses.SecondTest -->
      <class name="baseClasses.FirstTest">
        <test-method signature="first()[pri:0, instance:baseClasses.FirstTest@69e153c5]" started-at="2023-05-15T17:45:58 EDT" name="first" finished-at="2023-05-15T17:46:03 EDT" duration-ms="5011" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- first -->
      </class> <!-- baseClasses.FirstTest -->
      <class name="baseClasses.ThirdTest">
        <test-method signature="third()[pri:0, instance:baseClasses.ThirdTest@25ce9dc4]" started-at="2023-05-15T17:46:03 EDT" name="third" finished-at="2023-05-15T17:46:03 EDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- third -->
      </class> <!-- baseClasses.ThirdTest -->
    </test> <!-- Test4 -->
  </suite> <!-- Suite -->
</testng-results>
